src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,35 @@
+package codeu.chat.common;
+
+// why is this class in the util package? Can we combine the two ServerInfo classes in the
+// codeu.chat.common package?
+
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+
+import java.io.IOException;
+
+public final class ServerInfo {
+
+    public Time startTime;
+    private final static String SERVER_VERSION = "1.0.0";
+    public Uuid version;
+
+    public ServerInfo() throws IOException{
+        //Set the startTime equal to the Time value that the server was started
+        this.startTime = Time.now();
+        this.version = Uuid.parse(SERVER_VERSION);
+    }
+
+    public ServerInfo(Uuid version) throws IOException{
```

Comment:
You do not really need the two constructors with just one arg - please remove those.